model/trace/gen-ai.yaml

```
@@ -0,0 +1,63 @@
+groups:
+  - id: gen_ai.llm.request
+    type: span
+    brief: >
+      A request to an LLM is modeled as a span in a trace. The span name should be a low cardinality value representing the request made to an LLM, like the name of the API endpoint being called.
+    attributes:
+      - ref: gen_ai.llm.system
+        requirement_level: recommended
```

See comment on the markdown file
```suggestion
        requirement_level: required
```